### ISSUE TITLE
Peut-on mettre à jour le package YH pour YesWiki

### ISSUE THREAD

**Issue by jvayssiere** [YunoHost-Apps/yeswiki_ynh#25]
Bonjour,

Le dernier package YunoHost pour YesWiki contient la version doryphore 2020-11-30-4, alors que la version la plus récente est doryphore 2021-03-25-19. Serait-il possible de mettre le package à jour?

La mise à jour depuis l'interface de YesWiki n'est pas possible depuis YesWiki quand YesWiki est déployé dans YunoHost (bug ou fonctionnalité, je ne sais pas). Voir issue #24 .

Merci!

**Comment by nils-van-zuijlen** [comment]
Non, effectivement, il n'est pas possible de faire les mises à jour du cœur de YesWiki depuis lui-même, pour conserver une cohérence avec la version ynh. Les installations et mises à jour d'extensions et thèmes sont possibles par contre.

La mise à jour vers la version 2021-04-06-2 est actuellement sur `testing` et arrive bientôt.

**Comment by tr1p0p** [comment]
Pourquoi une fonction aussi essentiel est un fais aussi **important** n'est même pas préciser sur le README ?? Ou pendant l'installation ?

Du coup un seul bouton peux casser l’entièreté de yeswiki, aucuns avertissements ? Et on cette application as une étoile sur le catalogue ??

**Comment by tr1p0p** [comment]
Pour ceux qui aurait "casser" leur wiki en appuyant sur un simple bouton de mise à jour. Voici la ligne à mettre dans le terminal pour restaurer la version livrer avec ce paquet :

`sudo yunohost app upgrade yeswiki -u https://github.com/YunoHost-Apps/yeswiki_ynh --force`
